Index: src/test/resources/features/create_event.feature
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/resources/features/create_event.feature b/src/test/resources/features/create_event.feature
new file mode 100644
--- /dev/null	
+++ b/src/test/resources/features/create_event.feature	
@@ -0,0 +1,12 @@
+Feature: As a user, I should be able to create events by clicking on Event tab under Activity Stream
+
+
+  Background:
+    Given the user is logged in as "hr10@cybertekschool.com"
+    And the user clicks on Event
+
+  @wip
+  Scenario: Creating event with specific time and date
+    When the user selects start date, ending date, time and sets the time zone
+    And the user gives a name to event and clicks Send button
+    Then the user should be able to create the event
\ No newline at end of file
Index: src/test/java/com/azulcrm/step_definitions/CreateEventStepDefs.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/java/com/azulcrm/step_definitions/CreateEventStepDefs.java b/src/test/java/com/azulcrm/step_definitions/CreateEventStepDefs.java
new file mode 100644
--- /dev/null	
+++ b/src/test/java/com/azulcrm/step_definitions/CreateEventStepDefs.java	
@@ -0,0 +1,37 @@
+package com.azulcrm.step_definitions;
+
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+public class CreateEventStepDefs {
+
+    @Given("the user is logged in as {string}")
+    public void the_user_is_logged_in_as(String userName) {
+        LoginStepDefs loginStepDefs = new LoginStepDefs();
+        loginStepDefs.the_user_logged_in_as(userName);
+    }
+
+
+    @Given("the user clicks on Event")
+    public void the_user_clicks_on_Event() {
+
+    }
+
+    @When("the user selects start date, ending date, time and sets the time zone")
+    public void the_user_selects_start_date_ending_date_time_and_sets_the_time_zone() {
+
+    }
+
+    @When("the user gives a name to event and clicks Send button")
+    public void the_user_gives_a_name_to_event_and_clicks_Send_button() {
+
+    }
+
+    @Then("the user should be able to create the event")
+    public void the_user_should_be_able_to_create_the_event() {
+
+    }
+
+
+}
